src/scripts/components/foundations/Button.js

```diff
@@ -40,16 +69,38 @@ export const StyleAnchor = css`
   }};
 `
 
+export const ButtonStyle = css`
+  box-shadow: ${props => props.theme.colors.button.shadow};
+  cursor: ${props => (props.disabled ? 'initial' : 'pointer')};
+  font-size: ${props => (props.anchor ? null : props.theme.fonts.button)};
+  font-weight: ${props =>
+    props.anchor
+      ? props.theme.fonts.weight.regular
+      : props.theme.fonts.weight.bold};
+  margin: ${props => props.margin};
+  opacity: ${props => (props.disabled ? 0.5 : null)};
+  pointer-events: ${props => (props.disabled ? 'none' : null)};
+  text-transform: ${props => (props.anchor ? null : 'uppercase')};
+  user-select: ${props => (props.anchor ? null : 'none')};
+  text-rendering: geometricPrecision;
+`
+
 export const ButtonStyleHover = css`
   backface-visibility: hidden;
-  transition: opacity ${props => props.theme.animation.time.repaint};
+  transition: all 0.5s;
 
   &:focus,
   &:hover {
-    opacity: ${props => props.theme.animation.opacity.hover};
+    filter: brightness(1.3);
   }
 `
 
+export const ButtonColor = styled.button`
+  ${ButtonStyle} ${ButtonStyleHover} ${ButtonStyleBackground} border-radius: 2px;
+  color: ${props => props.theme.colors.button.white};
+  padding: 8px 18px;
+`
+
```

Comment:
Also, in general I think we may want to examine this component a bit. I tried to use it for tipping but found some bugs w/ the active/hover states that I could not easily work around. Essentially the button appeared in those states inappropriately after clicking on and off. 

Just an FYI (have you seen issues like this?)

Reply:
>have you seen issues like this?

Probably some `:active` selector. Could you send more info about this, please?